crates/router/src/core/webhooks/incoming.rs

```diff
@@ -135,7 +135,7 @@ async fn incoming_webhooks_core<W: types::OutgoingWebhookType>(
         1,
         &[metrics::KeyValue::new(
             MERCHANT_ID,
-            merchant_account.merchant_id.clone(),
+            merchant_account.get_id().get_string_repr().to_owned(),
```

Comment:
We can directly pass the merchant ID domain type here (and in other usages in this file)?